homeassistant/components/ollama/conversation.py

```diff
@@ -52,9 +52,6 @@ def __init__(self, entry: OllamaConfigEntry, subentry: ConfigSubentry) -> None:
     async def async_added_to_hass(self) -> None:
         """When entity is added to Home Assistant."""
         await super().async_added_to_hass()
-        assist_pipeline.async_migrate_engine(
-            self.hass, "conversation", self.entry.entry_id, self.entity_id
-        )
         conversation.async_set_agent(self.hass, self.entry, self)
```

Comment:
Also makes me wonder if we should drop the old "agent" approach. Guess we need to deprecate that first when a user uses that. Would be nice if we register entity ID when user uses agent reference and we can tell them what to do. Anyway for a different PR